src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/Mocks.java

```diff
@@ -187,7 +188,6 @@ public static OverseasEntitySubmissionDto buildSubmissionDtoWithBoCorporateTrust
 
     private static EntityDto buildEntityDto() {
         EntityDto entityDto = new EntityDto();
-        entityDto.setName("Joe Bloggs Ltd");
```

Comment:
Please revert this change - I've done this in my PR (https://github.com/companieshouse/overseas-entities-api/pull/157).

Reply:
Reverted